### PR TITLE
behave v1.2.6 (ursaverance effort)

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-2635] behave v1.2.6

dev url: https://github.com/behave/behave/tree/v1.2.6
conda-forge: https://github.com/conda-forge/behave-feedstock
dependencies: https://github.com/behave/behave/blob/v1.2.6/setup.py

- linter
- reset build number
- remove `noarch`
- python script

**Dependency for:**
- AnacondaRecipes/allure-python-feedstock#1

**Depends on**
- AnacondaRecipes/parse_type-feedstock#2
- AnacondaRecipes/parse-feedstock#2

[PKG-2635]: https://anaconda.atlassian.net/browse/PKG-2635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,23 +8,24 @@ package:
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: b9662327aa53294c1351b0a9c369093ccec1d21026f050c3bd9b3e5cccf81a86
 
 build:
-  number: 1004
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+  skip: true  # [py<37]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
not seen upstream

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,23 +8,24 @@ package:
...
-    - python >=3.6
+    - python
     - setuptools <58   # because of https://github.com/behave/behave/issues/955
                        # which will be released with 1.2.7
     - pip
+    - wheel
   run:
-    - python >=3.6
+    - python
     - parse >=1.6.3
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - parse >=1.8.2
```
https://github.com/behave/behave/blob/v1.2.6/setup.py#L71

File: recipe/meta.yaml [hunk 3]
```diff
@@ -9,23 +8,24 @@ package:
...
     - parse_type >=0.3.4
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - parse_type >=0.4.2
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -9,23 +8,24 @@ package:
...
     - six
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - six >=1.11
```

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**